packages/@react-native-windows/cli/src/runWindows/utils/autolink.ts

```diff
@@ -425,6 +425,16 @@ export class AutolinkWindows {
             'projects' in windowsDependency &&
             Array.isArray(windowsDependency.projects)
           ) {
+            if (
+              windowsDependency.projects.length === 0 &&
+              dependencyName.includes('react-native')
+            ) {
+              // the dependency is probably a react native module, but we didn't find a module project
```

Comment:
When we implement modules delivered via nuget (as specd) we'll only want to throw an error if both `projects` and `nugetPackages` is empty.